.github/workflows/release_notes.yml

```diff
@@ -0,0 +1,82 @@
+name: Create release notes changelog
+on:
+  release:
+    types: [created]
```

Comment:
Added a github actions that will do:

`ruby GithubIssueStats.rb > changelog.txt`

Then a python script that will locate the created release, and basically do `current_release.body += changelog`

This is untested since it will not run until it's on develop and I create a release, but I have tested it that the python script works fine.